src/types/any.rs

```diff
@@ -59,10 +66,449 @@ pyobject_native_type_convert!(
 pyobject_native_type_extract!(PyAny);
 
 impl PyAny {
+    /// Convert this PyAny to a concrete Python type.
     pub fn downcast<T>(&self) -> Result<&T, PyDowncastError>
     where
         for<'py> T: PyTryFrom<'py>,
     {
         <T as PyTryFrom>::try_from(self)
     }
+
```

Comment:
You removed `fn None`?
Though I'm for that since we can get None by `py.None()`, it should be added to the removed section in the CHANGELOG.

Reply:
Yeah I didn't think `ObjectProtocol::None` was ever a nice API 😄 .

I'll add detail to the `Changed` section where I explain the overall change.